src/background/services/openPayments.ts

```diff
@@ -1023,6 +1070,15 @@ export const isOutOfBalanceError = (error: any) => {
   return error.status === 403 && error.description === 'unauthorized';
 };
 
+export const isMissingGrantPermissionsError = (error: any) => {
+  if (!isOpenPaymentsClientError(error)) return false;
+  return (
+    error.status === 403 &&
+    (error.description === 'Insufficient Grant' /* Fynbos */ ||
```

Comment:
What does the `Fynbos` comment represent?

Reply:
Fynbos shows this error as "Insufficient grant", while test wallet fails with "Inactive token". Quite weird I know, but this check passes both.

Reply:
Resolution: since Rafiki alpha.16, we get "Inactive Token" only.